public/app/core/utils/kbn.ts

```diff
@@ -575,13 +575,15 @@ kbn.valueFormats.m3 = kbn.formatBuilders.fixedUnit('m3');
 kbn.valueFormats.Nm3 = kbn.formatBuilders.fixedUnit('Nm3');
 kbn.valueFormats.dm3 = kbn.formatBuilders.fixedUnit('dm3');
 kbn.valueFormats.gallons = kbn.formatBuilders.fixedUnit('gal');
-
+ 
```

Comment:
You have a trailing whitespace here that made the frontend test fail. Please remove